fenya123/acih-backend
# 7: Add unit-tests configuration

Parent story: https://github.com/fenya123/bingin/issues/10

Here at **fenya123** we _respect_ programming. And that is exactly why we want our code to be covered by tests. It isn't a necessity in any way: after all, it's not like any code produced at **fenya123** will ever require any amount of tuning or refactoring; it is our _respect_ to the craft that compels us to abide by industry standarts.

For testing purposes we will use `pytest`. Additional coverage-related functionality will be provided by `pytest-cov`.

In the scope of this task we need to:
1. install `pytest` and `pytest-cov` via `poetry`
2. create `pytest.ini` and `.coveragerc` files
3. configure coverage to fail if it is under 100%, to check branch coverage and allow it to exclude lines from coverage
4. write POC test and check that everything works as intended 
5. update `README.md` and `pyproject.toml` to accomodate changes
